### ISSUE TITLE
Example input files available for downloading?

### ISSUE THREAD

**Issue by maojn** [nanoporetech/tombo#330]
Hi,

Is there a set of example files that I can use to walk through your tutorial with? I do not work in the field and don't have access to example files to test the installation of tombo. Thank you.

**Comment by Chris-Kimmel** [comment]
Nope. Last year Dr. Stoiber said ONT was working to release example datasets, but I haven't heard anything since. See https://github.com/nanoporetech/tombo/issues/249#issuecomment-634768699

**Comment by marcus1487** [comment]
A small dataset has been hard to get to release. We will have a small dataset shipped with a new package (Tombo2) which should be released in the coming weeks.
